CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Version 1.1.0 (unreleased)
     In particular, a few `long` types were converted to `intptr_t` and `int64_t`
     where appropriate. In particular, this affects the `field` type that may be used
     by C++ end users (so they exist).
+*   Added new cursor.fetchnumpybatches method which returns a generator to
```

Comment:
please use backticks and brackets to mark this as a function: `cursor.fetchnumpybatches()`